### DISCUSSION TITLE
Client-side table column filters

### DISCUSSION THREAD

**Discussion by Quatters** [apache/superset#26364] [Ideas]
We are migrating from Grafana. Simple but handy feature, that we are missing when working with Table chart, is [column filters](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/table/#column-filter). Are there any plans to implement them? I'm interested if somebody else missing this feature too, because we develop fork of Superset and have implemented column filters by ourselves. Maybe I can make a PR (draft at least). This is how it looks in our implementation:

[Screencast from 27.12.2023 17:54:00.webm](https://github.com/apache/superset/assets/73882772/17be1003-e40a-4627-bf7c-4bc318bd7e36)

**Comment by aikawa-ohno** [comment]
I'm using Superset 3.1,
I don't think this feature has been implemented.
Personally, I would like this feature and I think
it would be very beneficial.

If possible, the following site
I want a column filter that can specify various conditions.

https://www.material-react-table.com/docs/guides/column-filtering#customize-filter-modes-demo
